docs/src/main/sphinx/release/release-450.md

```diff
@@ -0,0 +1,131 @@
+
+## Cassandra connector
+
+* Fix incorrect behavior that uses one more partition than expected when
+  specifying a value for the `cassandra.partition-size-for-batch-select`
+  configuration property. ({issue}`21940`)
+
+## ClickHouse connector
+
+* Improve performance of `ORDER BY ... LIMIT` on non-textual types by pushing
+  execution down to the underlying database. ({issue}`22174`)
+
+## Delta Lake connector
+
+* Add support for concurrent `UPDATE`, `MERGE`, and `DELETE` queries. ({issue}`21727`)
+* Add support for using table statistics with `TIMESTAMP` types. ({issue}`21878`)
+* Add support for reading tables with
+  [type widening](https://docs.delta.io/latest/delta-type-widening.html). ({issue}`21756`)
+* Set the default value for the `s3.max-connections` configuration property
+  to 500. ({issue}`22209`)
+* Fix failure when reading a `TIMESTAMP` value after the year 9999. ({issue}`22184`)
+* Fix failure when reading tables with the unsupported `variant` type. ({issue}`22310`)
+
+## Hive connector
+
+* Add support for `integer` to `varchar` and `decimal` to `varchar` type 
+  coercion in unpartitioned tables. ({issue}`22246`, {issue}`22293`)
+* Add support for `double` to `varchar` type coercion in unpartitioned tables
+  using Parquet files. ({issue}`22277`)
+* Add support for `float` to `varchar` type coercion. ({issue}`22291`)
+* Set the default value for the `s3.max-connections` configuration property
+  to 500. ({issue}`22209`)
+
+## Hudi connector
+
+* Set the default value for the `s3.max-connections` configuration property
+  to 500. ({issue}`22209`)
+
+## Iceberg connector
+
+* Add support for reading Iceberg
+  [UniForm](https://docs.delta.io/latest/delta-uniform.html) tables. ({issue}`22106`)
+* Add support for the `TRUNCATE` statement. ({issue}`22340`)
+* Add support for V2 of the Nessie REST API. Previous behavior can be
+  restored by setting the `iceberg.nessie-catalog.client-api-version`
+  configuration property to `V1`. ({issue}`22215`)
+* Improve performance when reading by populating `split_offsets` in file
+  metadata. ({issue}`9018`)
+* Set the default value for the `s3.max-connections` configuration property
+  to 500. ({issue}`22209`)
+* Fix failure when reading Parquet files that don't have `field-id` on
+  structured types. ({issue}`22347`)
+
+## MariaDB connector
+
+* Add support for [fault-tolerant execution](/admin/fault-tolerant-execution). ({issue}`22328`)
+* Improve performance of listing table columns. ({issue}`22241`)
+
+## Memory connector
+
+* Add support for the `TRUNCATE` statement. ({issue}`22337`)
+
+## MySQL connector
+
+* Improve performance of listing table columns. ({issue}`22241`)
+
+## Pinot connector
+
+* Add support for the
+  [`enableNullHandling` query option](https://docs.pinot.apache.org/developers/advanced/null-value-support#advanced-null-handling-support). ({issue}`22214`)
+* Fix failure when using dynamic tables. ({issue}`22301`)
```

Comment:
Those are Pinot dynamic tables I assume ... should we link to some docs or so (I couldnt find any quickly)